### PR TITLE
Usb-bus-on-raspberry-pi.adoc : correct indefinite article on problem

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**

### CODE REVIEW

**Review by lurch — APPROVED** [review]

File: documentation/asciidoc/computers/raspberry-pi/usb-bus-on-raspberry-pi.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ In general, every device supported by Linux can be used with a Raspberry Pi, alt
 
 === Maximum Power Output
 
-As with all computers, the USB ports on the Raspberry Pi supply a limited amount of power. Often problems with USB devices are caused by power issues. To rule out power as the cause of an problem, connect your USB devices to the Raspberry Pi using a powered hub.
+As with all computers, the USB ports on the Raspberry Pi supply a limited amount of power. Often problems with USB devices are caused by power issues. To rule out power as the cause of a problem, connect your USB devices to the Raspberry Pi using a powered hub.
```
Comment (aallan) [inline documentation/asciidoc/computers/raspberry-pi/usb-bus-on-raspberry-pi.adoc:7]: Should be "the problem" not "a" or "an".

Comment (lurch) [reply documentation/asciidoc/computers/raspberry-pi/usb-bus-on-raspberry-pi.adoc:7]: I guess you could also use "To rule out insufficient power as..." ?

Comment (aallan) [reply documentation/asciidoc/computers/raspberry-pi/usb-bus-on-raspberry-pi.adoc:7]: Yup, that would be better.

Comment (RuffaloLavoisier) [reply documentation/asciidoc/computers/raspberry-pi/usb-bus-on-raspberry-pi.adoc]: @lurch @aallan 
Thanks advice.

**Review by lurch — APPROVED** [review]

**Review by aallan — APPROVED** [review]

**Pull request merged**